src/transformer/encode.ts

```diff
@@ -671,7 +692,7 @@ const encoders: NodeEncoderMap = {
       inlineContentsOfNodeType(node, node.type.schema.nodes.section_label) ||
       undefined,
     path: path.concat([node.attrs.id]),
-    elementIDs: childElements(node)
+    elementIDs: sectionChildElements(node)
       .map((childNode) => childNode.attrs.id)
       .filter((id) => id),
```

Comment:
Can we move these into the method? We can rename it to `sectionChildElementIds`.